src/suggesters/completion-suggester.js

```diff
@@ -57,6 +57,19 @@ class CompletionSuggester extends Suggester {
         return this;
     }
 
+    /**
+     * Sets whether duplicate suggestions should be filtered out (defaults to false).
+     *
+     * NOTE: This option was added in elasticsearch v6.1.
+     *
+     * @param {boolean} skip Enable/disable skipping duplicates
+     * @returns {CompletionSuggester} returns `this` so that calls can be chained.
+     */
+    skipDuplicates(skip = true) {
```

Comment:
The doc comment seems to conflict with the default value here.

Reply:
Would prefer to avoid the default parameter value.

Reply:
The doc comment means that it's false if you don't call this method. I was following the example of `fuzzy(fuzzy = true)` and also thinking about what the caller would read:

```suggester.skipDuplicates()```

seems clear, while

```suggester.skipDuplicates(true)```

seems a little redundant. While it's unlikely anyone would ever need `skipDuplicates(false)`, it's there just in case someone has a common code module that enables it but then needs to disable it.

All that said, I'm okay changing it if you don't find those arguments persuasive. Just let me know.